planner/core/integration_test.go

```diff
@@ -3205,31 +3205,40 @@ func TestLimitPushDown(t *testing.T) {
 
 	tk.MustExec(`create table t (a int)`)
 	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
+	tk.MustExec(`insert into t values (1)`)
 	tk.MustExec(`analyze table t`)
 
 	tk.MustExec(`set tidb_opt_limit_push_down_threshold=0`)
 	tk.MustQuery(`explain format=brief select a from t order by a desc limit 10`).Check(testkit.Rows(
```

Comment:
better set the LIMIT bigN>10 here?

Reply:
Actually, this test is not for this PR... This test is used to test whether `tidb_opt_limit_push_down_threshold` and `limit_to_cop()` can take effect.

Reply:
get